### ISSUE TITLE
Have TensorBoard graph smoothing not overestimate last gotten scalar

### ISSUE THREAD

**Issue by carlthome** [tensorflow/tensorflow#6439]
When plotting scalars in TensorBoard, the smoothed curve overfits the latest value. It means the curve jumps around with subsequent updates, and for a line fit it just plainly looks bad and feels wrong.

For example, when progress reporting neural networks training and plotting the loss over the training data, surely the blue line will continue with a slight downwards slope given a stable learning rate but instead this happens at the end of the curve:

![image](https://cloud.githubusercontent.com/assets/1595907/21398836/3efff694-c7a9-11e6-94cb-1b996021c1d4.png)

![image](https://cloud.githubusercontent.com/assets/1595907/21398869/64c3fe66-c7a9-11e6-8440-29ca1e959d91.png)

**Comment by aselle** [comment]
@danmane, could you please take a look?

**Comment by cancan101** [comment]
Is this fixed by: https://github.com/tensorflow/tensorflow/pull/7891 ?

**Comment by carlthome** [comment]
https://github.com/tensorflow/tensorflow/pull/7891 looks great.

**Comment by carlthome** [comment]
Fixed?

**Comment by carlthome** [comment]
Fixed in 1.1
